scripts/AGENTS.md

```
@@ -0,0 +1,71 @@
+# Weekly (Monday 06:00 UTC) - updates race results
+python scripts/update_historical.py
+
+# January yearly - updates season calendar  
+python scripts/update_seasons.py --years 2026,2027
+```
+
+**Output**: Commits updated JSON to `app/assets/seasons/` and `app/assets/circuits_data.json`.
+
+## Asset Processing Pipeline
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)